### DISCUSSION TITLE
Report: tsync error on LineageOS 18.1

### DISCUSSION THREAD

**Discussion by ramimrashid** [ROM-builders/temporary#22022] [General]
I've noticed that most of the people including me who wants to build lineageOS 18.1 have faced tsync error. I've tried many times but every time faced tsync error. 

Here's the link of my device (google pixel 4xl) - https://cirrus-ci.com/task/5004709879087104

Here's the device of someone else (poco f1) - https://cirrus-ci.com/build/5484325588697088

**Comment by ramimrashid** [comment]
@Apon77, there's still tsync error on lineageos 18.1

Error: https://cirrus-ci.com/task/6223930969554944

**Comment by Apon77** [comment]
No idea why this happened. dir is removed. Hopefully it will be fine now
